internal/plugin/manager/nvml.go

```diff
@@ -34,7 +34,7 @@ func (m *nvmlmanager) GetPlugins() ([]plugin.Interface, error) {
 
 	var plugins []plugin.Interface
 	for _, r := range rms {
-		plugin, err := plugin.NewNvidiaDevicePlugin(m.config, m.kubeletSocket, r, m.cdiHandler)
+		plugin, err := plugin.NewNvidiaDevicePlugin(m.config, m.kubeletSocket, r, m.cdiHandler, m.imexChannels)
```

Comment:
Same comment as before applies here -- this extra flag seems out of place, but let's iterate on a future PR to figure out how best to encapsulate this.

Reply:
If you feel strongly about it, I can refactor things before this change, otherwise I think we can clean it up as a follow-up.

Reply:
See https://github.com/NVIDIA/k8s-device-plugin/pull/991